app/src/main/java/mozilla/lockbox/action/SettingAction.kt

```diff
@@ -44,6 +46,24 @@ class Setting {
                     Never -> R.string.never
                 }
             }
+
+        val seconds: Int by lazy {
+            // I'm concerned about this method of setting time. I wonder if it's going
+            // to be circumventable by changing the device time. we might want to consider a
+            // options for mitigating that attack vector.
```

Comment:
We should probably use [`SystemClock.elapsedRealtime()`][0] instead of translating to a date-time. If we are thinking about `AlarmManager`, then we should double check that the calculate `elapsedRealtime` actually has passed.

I'm not sure what the status of the designs are, but we should definitely force the user to unlock after a reboot.

[0]: https://developer.android.com/reference/android/os/SystemClock.html

Reply:
+1. Have explored the `SystemClock.elapsedRealtime()` option and I think that's the way to go.

@changecourse @nickbrandt are y'all OK with the app locking if the device has rebooted, regardless of time elapsed?

Reply:
@sashei yea that makes sense to me 👍